from pydantic import BaseModel

class InvalidInputError(Exception):
    """Erro personalizado para entradas inv치lidas no c치lculo."""
    def __init__(self, message="Entrada inv치lida fornecida. Por favor, revise os par칙metros."):
        self.message = message
        super().__init__(self.message)
